front/.eslintrc.js

```diff
@@ -25,14 +18,17 @@
     */
     "unused-imports/no-unused-imports": "error",
     "react/no-unescaped-entities": 0,
+    "@typescript-eslint/no-explicit-any": 0,
+    "no-case-declarations": 0,
```

Comment:
This rule forces you to handle switch cases in curly bracket, because otherwise the scope of the objects declared in it is global to the switch, which can cause surprising behavior.
To complicated to fix for now, lets do it later.